src/tools/run-make-support/src/lib.rs

```diff
@@ -115,3 +115,99 @@ fn handle_failed_output(cmd: &Command, output: Output, caller_line_number: u32)
     eprintln!("=== STDERR ===\n{}\n\n", String::from_utf8(output.stderr).unwrap());
     std::process::exit(1)
 }
+
+macro_rules! impl_common_helpers {
```

Comment:
is it worth documenting (or just commenting) what is delegated for developers at least?

Reply:
I'll make sure to add some docs

Reply:
Thanks for the reminder, I added some docs to explain what this macro provides.